sdk/keyvault/keyvault-keys/test/internal/crypto.spec.ts

```diff
@@ -143,7 +143,10 @@ describe("internal crypto tests", () => {
         sinon.assert.calledWith(
           encryptStub,
           { algorithm: "RSA1_5", plaintext: text },
-          { requestOptions: { timeout: 5 }, tracingOptions: { spanOptions: {} } }
+          operationOptionsSinonMatcher({
```

Comment:
I think I mentioned this but it got lost in the shuffle because I can't find it anymore. This is fine, but you're welcome to use something like

```typescript
        sinon.assert.calledWith(
          encryptStub,
          { algorithm: "RSA1_5", plaintext: text },
          sinon.match({ requestOptions: { timeout: 5 }})
        );

```

Which lets us be more lax about matching, since we only care that the provided timeout parameters get mapped correctly.

Reply:
I think what I want to do is move this sinon matcher (and maybe even the test methodlogy) into a common package (although not in this PR since just merging this PR is taking forever).

Reply:
I'll file an issue for this. I think what we've got in there now is probably fine but it's nice to test that parenting propagated properly.